bin/oneoff/diff_email_previews

```diff
@@ -16,21 +16,22 @@ To diff emails in 2 folders in the current directory: './diff_email_previews dif
 }
 
 # List of email previews are copied from
-# http://localhost-studio.code.org:3000/rails/mailers/pd/workshop_mailer/
+# http://localhost-studio.code.org:3000/rails/mailers/pd_workshop_mailer/
 PREVIEWS = %w(
```

Comment:
This whole list I re-copied and pasted from my localhost. I'm not sure how often we use this, but figured I'd update it